core/services/beacon/default-settings.json

```diff
@@ -15,7 +15,7 @@
       },
       {
         "name": "wlan0",
-        "domain_names": ["companion", "blueos", "blueos-wifi"],
```

Comment:
`companion` is used temporarily for retrocompatibility and should be removed soon.
`companion.local` is also redirected to `blueos.local` to 'teache' the user that the new address should be used. The patch should only remove "blueos" from the wifi publishment.